test/test_c10d.py

```diff
@@ -974,6 +976,7 @@ def _test_scatter_stress(self, inputs, fn):
             self.assertEqual(
                 torch.Tensor([iter + root]),
                 outputs[iter][root],
+                None,
```

Comment:
Will it be more clear if we use `message="Mismatch in iteration %d for rank %d" % (iter, root)` here instead of setting `prec`?

Reply:
+1 -- this is what I'm doing in #25260 as well. Since I'll have to rebase there anyway, I'll fix this up there.

Reply:
i've done this in #25557 as well